packages/speech/src/index.js

```diff
@@ -137,6 +137,7 @@ Speech.prototype.endpointerTypes = {
  */
 Speech.detectEncoding_ = function(filename) {
   if (!is.string(filename)) {
+    // istanbul ignore next
```

Comment:
Oops, did this drop our coverage? I'd rather just add a test for this. I can do this in a separate PR.